gprofiler/metadata/system_metadata.py

```diff
@@ -194,15 +205,39 @@ class SystemInfo:
     spawn_uptime_ms: float
 
 
+def get_windows_network_details() -> Tuple[str, str]:
+    IPV4_PATTERN = r"^(?:[0-9]{1,3}\.){3}[0-9]{1,3}$"
+    LOOPBACK = "127.0.0.1"
+    ADDR_KEY = "addr"
+    MAC_KEY = -1000
+    for iface in netifaces.interfaces():
+        mac_address = None
+        for key, addresses in netifaces.ifaddresses(iface).items():
+            for address in addresses:
+                if key == MAC_KEY:
+                    mac_address = address[ADDR_KEY]
+                    continue
+                if re.match(IPV4_PATTERN, address[ADDR_KEY]) is not None and address[ADDR_KEY] != LOOPBACK:
+                    return cast(Tuple[str, str], (mac_address, address[ADDR_KEY]))
+    return UNKNOWN_VALUE, UNKNOWN_VALUE
+
+
 def get_static_system_info() -> SystemInfo:
-    hostname, distribution, libc_tuple, mac_address, local_ip = _initialize_system_info()
-    clock = getattr(time, "CLOCK_BOOTTIME", time.CLOCK_MONOTONIC)
-    try:
-        spawn_uptime_ms = time.clock_gettime(clock)
-    except OSError as error:
-        if error.errno != errno.EINVAL:
-            raise
-        spawn_uptime_ms = time.clock_gettime(time.CLOCK_MONOTONIC)
+    if is_windows():
+        hostname = platform.node()
+        distribution = platform.system(), platform.release(), platform.version()
```

Comment:
What does it return on Windows?

Reply:
platform.node() returns the hostname of your machine
platform.system() returns "Windows"
platform.release() returns '10'
platform.version() returns '10.0.19044'